product_docs/docs/biganimal/release/reference/cli.mdx

```diff
@@ -55,24 +55,34 @@ press [Enter] to continue in the web browser...
 Credential "ba-user1" created!
 ```
 
+Refresh tokens expire after 30 days. To continue using the credential to access the CLI, use the `reset-credential` command to authenticate through the BigAnimal website and receive a new refresh token.
+
+```shell
+$ biganimal reset-credential ba-user1 
+Visit this URL to login: https://auth.biganimal.com/activate?user_code=****-****
+or press [Enter] to continue in the web browser 
+
+Credential "ba-user1" reset operation succeeded
+
+```
+
 You can create multiple credentials for different BigAnimal accounts and then set one as context of your current management session. Use `show-credentials` to list all available credentials and use `config set context_credential` to set a default credential for the current context. For example:
 
 ```
 $ biganimal show-credentials
-┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ Credentials                                                                                         ┃
-┣━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━┫
-┃ Name     ┃ Address                     ┃ Port ┃ TokenType ┃ AccessToken ┃ RefreshToken ┃ IdToken    ┃
-┣━━━━━━━━━━╋━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━━╋━━━━━━━━━━━╋━━━━━━━━━━━━━╋━━━━━━━━━━━━━━╋━━━━━━━━━━━━┫
-┃ ba-user1 ┃ portal.biganimal.com ┃ 443  ┃ Bearer           ┃ **********  ┃ **********   ┃ ********** ┃
-┃ ba-user2 ┃ portal.biganimal.com ┃ 443  ┃ Bearer           ┃ **********  ┃ **********   ┃ ********** ┃
-┗━━━━━━━━━━┻━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━━┻━━━━━━━━━━━┻━━━━━━━━━━━━━┻━━━━━━━━━━━━━━┻━━━━━━━━━━━━┛
-$ biganimal config set context_credential ba-user1
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃ Credentials                                                             ┃
+┣━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━┫
+┃ Name     ┃ Address                     ┃ Port ┃  Context credential     ┃ 
+┃ ba-user1 ┃ portal.biganimal.com        ┃ 443  ┃ x                       ┃
+┃ ba-user2 ┃ portal.biganimal.com        ┃ 443  ┃                         ┃
+┗━━━━━━━━━━┻━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━━┻━━━━━━━━━━━━━━━━━━━━━━━━━┛
```

Comment:
@moiznalwalla , where did you got this output? this is not what CLI's real outputs.

Reply:
This is what I got when i ran show-credentials. not sure what might have gone wrong.

![image](https://user-images.githubusercontent.com/90263457/181229382-ce21489a-b3af-4791-91a6-fe83f3f6a8d5.png)

Reply:
We also had a discussion about the change in output in slack.
<img width="815" alt="image" src="https://user-images.githubusercontent.com/90263457/181229645-9d36e075-c42f-44c7-b229-c9cccfb96089.png">

Reply:
@moiznalwalla this is what you committed.
<img width="520" alt="image" src="https://user-images.githubusercontent.com/5901001/181236625-100fd572-446b-4c6c-b8ab-73406b17d9c1.png">

This is what your command outputs
![image](https://user-images.githubusercontent.com/5901001/181236642-2d0da1c9-d33f-4adf-ae72-fbb33236657e.png)

Are they the same?

Reply:
Fixed the casing and formatting in the code block as close as possible to output.